python/xorbits/_mars/dataframe/datasource/read_parquet.py

```diff
@@ -395,33 +397,52 @@ def _tile_no_partitioned(cls, op: "DataFrameReadParquet"):
 
         dtypes = cls._to_arrow_dtypes(out_df.dtypes, op)
         shape = (np.nan, out_df.shape[1])
-
         path_prefix = ""
+        z = None
         if isinstance(op.path, (tuple, list)):
             paths = op.path
         elif get_fs(op.path, op.storage_options).isdir(op.path):
             parsed_path = urlparse(op.path)
             if parsed_path.scheme.lower() == "hdfs":
                 path_prefix = f"{parsed_path.scheme}://{parsed_path.netloc}"
             paths = get_fs(op.path, op.storage_options).ls(op.path)
+        elif isinstance(op.path, str) and op.path.endswith(".zip"):
+            z = zipfile.ZipFile(op.path)
```

Comment:
Can you support reading parquets from a remote url with a zip suffix? e.g. `http://127.0.0.1:8000/my_package.zip`?